### PR TITLE
EZP-24929 eng-GB hardcoded in the preview URI

### PR DESCRIPTION
**Pull request by StephaneDiot**
jira : https://jira.ez.no/browse/EZP-24929

## Description

The language code was hardcoded in the URI, now the language code is given to the editPreviewView

## Tests

- [x] Units tests

### CODE REVIEW

File: Resources/public/js/views/actions/ez-previewactionview.js [hunk 1]
```diff
@@ -154,6 +154,21 @@ YUI.add('ez-previewactionview', function (Y) {
...
+
+            /**
+             * The languageCode
+             *
+             * @attribute languageCode
+             * @type {String}
+             * @default {}
+             */
+            languageCode: {
+                value: {},
```
Comment (dpobel) [inline Resources/public/js/views/actions/ez-previewactionview.js]: `@type {String}` but default value to `{}` ? default to `''` would be more suited

File: Resources/public/js/views/ez-editactionbarview.js [hunk 2]
```diff
@@ -99,6 +113,19 @@ YUI.add('ez-editactionbarview', function (Y) {
...
+            },
+
+            /**
+             * The language code of the content currently edited
+             *
+             * @attribute languageCode
+             * @type {String}
+             */
+            languageCode: {
+                value: {},
```
Comment (dpobel) [inline Resources/public/js/views/ez-editactionbarview.js]: same as https://github.com/ezsystems/PlatformUIBundle/pull/385/files#r42241825 (and missing the @default tag in the JS Doc)

File: Resources/public/js/views/ez-editpreviewview.js [hunk 3]
```diff
@@ -152,7 +152,19 @@ YUI.add('ez-editpreviewview', function (Y) {
...
+
+            /**
+             * The languageCode of the content previewed
+             *
+             * @attribute languageCode
+             * @type String
+             * @required
+             */
+            languageCode: {
+                value: {}
```
Comment (dpobel) [inline Resources/public/js/views/ez-editpreviewview.js]: and here as well

File: Tests/js/views/actions/assets/ez-previewactionview-tests.js [hunk 4]
```diff
@@ -85,6 +85,7 @@ YUI.add('ez-previewactionview-tests', function (Y) {
         setUp: function () {
             this.editPreview = new Y.Mock();
             this.contentMock = new Y.Mock();
+            this.languageCodeMock = new Y.Mock();
```
Comment (dpobel) [inline Tests/js/views/actions/assets/ez-previewactionview-tests.js]: a mock of a string ?

**Comment by StephaneDiot** [comment]
ready for review @dpobel @yannickroger @mhyndle

**Comment by dpobel** [comment]
+1 once Travis agrees

**Comment by mhyndle** [comment]
+1

**Comment by StephaneDiot** [comment]
what about travis fail ?

**Pull request merged**